src/utils/accessibility/accessibility.js

```diff
@@ -3,7 +3,8 @@ const PIXELS_IN_POINT = 1.333;
 
 // http://www.w3.org/TR/2008/REC-WCAG20-20081211/#larger-scaledef
 function isLargeScale(fontSize, fontSizeUnit, isFontBold) {
-  return isFontBold ? (fontSizeInPt(fontSize, fontSizeUnit) >= 14) : (fontSizeInPt(fontSize, fontSizeUnit) >= 18);
+  const points = fontSizeInPt(fontSize, fontSizeUnit);
+  return isFontBold ? (points >= 14) : (points >= 18);
```

Comment:
Make sure to run `npm run build` before committing. This will run the linter and the tests. (Now the build is broken because of a lint issue.)